contracts/JBBuybackDelegate.sol

```diff
@@ -190,7 +190,7 @@ contract JBBuybackDelegate is ERC165, JBOperatable, IJBBuybackDelegate {
             delegateAllocations[0] = JBPayDelegateAllocation3_1_1({
                 delegate: IJBPayDelegate3_1_1(this),
                 amount: _amountToSwapWith,
-                metadata: abi.encode(_quoteExists, _projectTokenIs0, _minimumSwapAmountOut, _data.weight)
+                metadata: abi.encode(_quoteExists, _projectTokenIs0, _minimumSwapAmountOut, _data.amount.value - _amountToSwapWith, _data.weight)
```

Comment:
Thoughts on calculating in didPay() instead of passing?

as:
```solidity
_data.amount.value - _data.forwardedAmount.value
```

Reply:
doing it in data source allows more explicit control for other data sources who wanna call upon the BBD's didPay, but pass a different amount to mint.